apps/jupyterlab/requirements.txt

```diff
@@ -0,0 +1 @@
+fastmcp
```

Comment:
what about things for openclip in embeddings?

Reply:
The notebook does not do the embedding.